File: sdk/src/main/java/com/sygic/travel/sdk/model/media/Original.java
package com.sygic.travel.sdk.model.media;

import com.google.gson.annotations.SerializedName;

/**
 * <p>Medium's original parameters.</p>
 */
class Original {
	private static final String SIZE = "size";
	private static final String WIDTH = "width";
	private static final String HEIGHT = "height";

	@SerializedName(SIZE)
	private int size;

	@SerializedName(WIDTH)
	private int width;

	@SerializedName(HEIGHT)
	private int height;

	public Original() {
	}

	public int getSize() {
		return size;
	}

	public void setSize(int size) {
		this.size = size;
	}

	public int getWidth() {
		return width;
	}

	public void setWidth(int width) {
		this.width = width;
	}

	public int getHeight() {
		return height;
	}

	public void setHeight(int height) {
		this.height = height;
	}
}

File: sdk/src/main/java/com/sygic/travel/sdk/model/media/Original.kt
package com.sygic.travel.sdk.model.media

import com.google.gson.annotations.SerializedName

/**
 * Medium's original parameters.
 */
class Original {

    @SerializedName("size")
    var size: Int = 0

    @SerializedName("width")
    var width: Int = 0

    @SerializedName("height")
    var height: Int = 0

}
